_posts/2018-10-09-enabling-ecc-on-pc-engines-apu2.md

```diff
@@ -0,0 +1,354 @@
+---
+post_title: Enabling ECC on PC Engines platforms
+author: Krystian Hebel
+layout: post
+published: true
+post_date: 2018-10-09 15:00:00
+
+tags:
+    - ECC
+    - APU2
+categories:
+    - Firmware
+
+---
+
+## Introduction
```

Comment:
@krystian-hebel we should describe at high level what we want to present in following blog post to let the reader know what to expect.

E.g.
1. How to test ECC and what problems we may face with that

Reply:
2. How to use Memtest86 Pro to inject ECC errors

Reply:
@pietrushnic  done